CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+
+If you are ready to contribute, please search for open issues labeled "help-wanted" and choose one to work on. Feel free to join our Slack channel to participate in discussions or create a new issue if you have a suggestion or request. When submitting a pull request, please reference the corresponding open issue if one exists. To automatically close related issues in GitHub, include keywords in your pull request descriptions and commit messages.
+
+**Contents**
+
+- <a name="contributing-flow">Contribution Flow</a>
+  - <a href="#commit-signoff">Developer Certificate of Origin</a>
+- WasmEdge Contribution Flow
+  - <a href="#contributing-docs">WasmEdge Docs</a> 
+  - <a href="https://wasmedge.org/book/en/contribute/contribute.html">WasmEdge Runtime</a>
+  - <a href="https://github.com/WasmEdge/WasmEdge/blob/master/SECURITY.md">Reporting a vulnerability</a>
+
+# <a name="contributing-flow">Contribution Flow</a>
+
+To contribute to WasmEdge Docs
+
+## <a name="commit-signoff">Signing-off on Commits (Developer Certificate of Origin)</a>
+
+To contribute to this project, you must agree to the Developer Certificate of
+Origin (DCO) for each commit you make. The DCO is a simple statement that you,
+as a contributor, have the legal right to make the contribution.
+
+See the [DCO](https://developercertificate.org) file for the full text of what you must agree to
+and how it works [here](https://github.com/probot/dco#how-it-works).
+To signify that you agree to the DCO for contributions, you simply add a line to each of your
+git commit messages:
+
+```
+Signed-off-by: John Doe <john.doe@example.com>
+```
+
+In most cases, you can add this signoff to your commit automatically with the
+`-s` or `--signoff` flag to `git commit`. You must use your real name and a reachable email
+address (sorry, no pseudonyms or anonymous contributions). An example of signing off on a commit:
+
+```
+$ commit -s -m “my commit message w/signoff”
+```
+
+To ensure all your commits are signed, you may choose to add this alias to your global `.gitconfig`:
+
+_~/.gitconfig_
+
+```
+[alias]
+  amend = commit -s --amend
+  cm = commit -s -m
+  commit = commit -s
+```
+
+Or you may configure your IDE, for example, Visual Studio Code to automatically sign-off commits for you:
+
+<a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>
+
+## <a name="contributing-docs">WasmEdge Docs</a> 
+
+If you want to contribute to the WasmEdge Docs, you can follow these steps:
+
+1. **Fork the project:** Click the "Fork" button on the top-right corner of the repository page to create a copy of the project in your GitHub account.
+
+2. **Clone the forked project:** Open a terminal or command prompt and type the following command:
+
+```
+git clone <forked-repository-url>
```

Comment:
Can we just say `git clone https://github.com/WasmEdge/docs.git`? In my mind, the contribution flow here just describes how to contribute to WasmEdge docs. The contribution flow for WasmEdge runtime will be much difficult.  I'd love to hear what you think.

Reply:
Yes, it makes complete sense with respect to WasmEdge Runtime, but for this line, we are referring to the user's forked repository URL, and adding the `https://github.com/WasmEdge/docs.git` IMO wouldn't make much sense here. But please feel free to let me know if this is something the team wants and I will gladly send a commit to fix it :)